### ISSUE TITLE
Default Accept header to JSON

### ISSUE THREAD

**Issue by tony-gutierrez** [ciaranj/node-oauth#350]
It would probably be nice to add a default header:

Accept: 'application/json, text/json'

so that you will almost always get JSON back.

**Comment by tanmaiaccion** [comment]
+1

**Comment by tanmaiaccion** [comment]
@tony-gutierrez check this workaround, it works.
https://github.com/ciaranj/node-oauth/pull/312#issuecomment-359614640

**Comment by sunnysmane** [comment]
Yes, it will be good if content type option is available in GET method as it is available in PUT and POST method.
